test/unit/src/common/components/select-repository-list/t_select-repository-list.js

```diff
@@ -79,12 +79,6 @@ describe('<SelectRepositoryListComponent /> instance', function() {
       spy.restore();
     });
 
-    it('should show spinner when repositories are being fetched', function() {
```

Comment:
We don't show spinner immediately when component is created, so this test is not needed anymore.